py3-grpcio-gcp.yaml

```diff
@@ -1,38 +1,90 @@
-# Generated from https://pypi.org/project/grpcio-gcp/
 package:
   name: py3-grpcio-gcp
   version: 0.2.2
-  epoch: 2
+  epoch: 3
   description: gRPC extensions for Google Cloud Platform
   copyright:
     - license: Apache-2.0
   dependencies:
-    runtime:
-      - py3-grpcio
-      - python-3
+    provider-priority: 0
+
+vars:
+  pypi-package: grpcio-gcp
+  import: grpc_gcp
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '300'
 
 environment:
   contents:
     packages:
-      - build-base
-      - busybox
-      - ca-certificates-bundle
-      - py3-setuptools
-      - python-3
-      - wolfi-base
+      - py3-grpcio-tools
+      - py3-supported-build-base
+      - py3-supported-setuptools
 
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      expected-sha256: e292605effc7da39b7a8734c719afb12ec4b5362add3528d8afad3aa3aa9057c
-      uri: https://files.pythonhosted.org/packages/source/g/grpcio-gcp/grpcio-gcp-${{package.version}}.tar.gz
+      repository: https://github.com/GoogleCloudPlatform/grpc-gcp-python
+      tag: v${{package.version}}
+      expected-commit: 8ef890243664ebd95ae0225cdf87e7442b570f9e
 
-  - name: Python Build
-    uses: python/build-wheel
+  - runs: |
+      cd src
+      # From src/setup.sh
+      cp -f ../template/version.py version.py
+      python3 -m grpc_tools.protoc -I. --python_out=grpc_gcp/proto grpc_gcp.proto
```

Comment:
this is really weird, because the only reason you have `python3` is because `py3-grpcio-tools` has a manually added runtime dependency on it.

so if you multi-version py3-grpcio-tools, it will break this.

Reply:
Well noticed, I'll fix that :)

Reply:
https://github.com/wolfi-dev/os/pull/34780/commits/b80c0466c790ce7a792608dc13f914ea10363885